apps/web/pages/organizations/[organizationId]/components/invoiceTable.tsx

```diff
@@ -8,6 +7,9 @@ export const InvoiceFragment = graphql(`
     id
     invoiceDate
     customerName
+    organization {
```

Comment:
will not work imho because if the org does not have any invoices you cannot create one. I think you need to give the orgId as param into InvoiceTable component.
Am I wrong?